doc/source/dev/development_environment.rst

```diff
@@ -214,10 +214,20 @@ Another frequently asked question is "How do I debug C code inside NumPy?".
 The easiest way to do this is to first write a Python script that invokes the C
 code whose execution you want to debug. For instance ``mytest.py``::
 
-    from numpy import linspace
+    import numpy as np
     x = np.arange(5)
     np.empty_like(x)
 
+The next thing to do is to obtain the correct version of for gdb(often the default
```

Comment:
It might make sense to include the instructions on checking that you have a debugger installed on the system *before* the above sentence. For example:

Another frequently asked question is "How do I debug C code inside NumPy?".
First you must ensure you have a debugger installed... <rest of the explanation>.

Reply:
you are right, environment should configured before writing a debugger file.